platform/darwin/scripts/style-spec-cocoa-conventions-v8.json

```diff
@@ -18,7 +18,8 @@
     "text-max-width": "maximum-text-width",
     "text-optional": "is-text-optional",
     "text-rotate": "text-rotation",
-    "text-size": "text-font-size"
+    "text-size": "text-font-size",
+    "text-writing-mode": "text-writing-modes"
```

Comment:
`make style-code` should’ve added an entry about this renamed property to the “Information for Style Authors” guide too.

Reply:
`make darwin-style-code` generated guides